user_manual/files/access_webdav.rst

```diff
@@ -58,6 +58,1028 @@ You can access files in Linux operating systems using the following methods.
+          </d:prop>
+          <d:status>HTTP/1.1 200 OK</d:status>
+        </d:propstat>
+      </d:response>
+      <d:response>
+        <d:href>/nextcloud/remote.php/dav/files/USERNAME/welcome.txt</d:href>
+        <d:propstat>
+          <d:prop>
+            <d:getlastmodified>Tue, 13 Oct 2015 17:07:35 GMT</d:getlastmodified>
+            <d:getcontentlength>163</d:getcontentlength>
+            <d:resourcetype/>
+            <d:getetag>"47465fae667b2d0fee154f5e17d1f0f1"</d:getetag>
+            <d:getcontenttype>text/plain</d:getcontenttype>
+          </d:prop>
+          <d:status>HTTP/1.1 200 OK</d:status>
+        </d:propstat>
+      </d:response>
+    </d:multistatus>
+
+
+.. _KB2668751: https://support.microsoft.com/kb/2668751
+.. _KB2123563: https://support.microsoft.com/kb/2123563
+.. _WebDAV Navigator: http://seanashton.net/webdav/
+.. _Android devices: https://play.google.com/store/apps/details?id=com.schimera.webdavnavlite
+.. _iPhones: https://itunes.apple.com/app/webdav-navigator/id382551345
+.. _BlackBerry devices: http://appworld.blackberry.com/webstore/content/46816
+======================================
+Accessing Nextcloud files using WebDAV
+======================================
+
+Nextcloud fully supports the WebDAV protocol, and you can connect and synchronize
+with your Nextcloud files over WebDAV. In this chapter you will learn how to
+connect Linux, macOS, Windows, and mobile devices to your Nextcloud server via
+WebDAV. Before we get into configuring WebDAV, let's take a quick look at the
+recommended way of connecting client devices to your Nextcloud servers.
+
+.. note:: In the following examples, you should replace **example.com/nextcloud** with the
+   URL of your Nextcloud server (omit the directory part if the installation is
+   in the root of your domain), and "USERNAME" with the username of the connecting user.
+
+   See the webdav url (bottom left, settings) on your Nextcloud.
+
+Nextcloud Desktop and mobile clients
+------------------------------------
+
+The recommended way to synchronize a desktop PC with a Nextcloud server is by
+using `Nextcloud/ownCloud sync clients <https://nextcloud.com/install/#install-clients>`_. You can configure the
+client to save files in any local directory and you can choose which
+directories on the Nextcloud server to sync with. The client displays the
+current connection status and logs all activity, so you always know which
+remote files have been downloaded to your PC and you can verify that files
+created and updated on your local PC are properly synchronized with the server.
+
+The recommended way to synchronize Nextcloud server with Android and
+Apple iOS devices is by using the `mobile apps
+<https://nextcloud.com/install/>`_.
+
+To connect your mobile app to a Nextcloud server use the
+base URL and folder only::
+
+    example.com/nextcloud
+
+In addition to the mobile apps provided by Nextcloud or ownCloud, you can use other apps to
```

Comment:
```suggestion
In addition to the mobile apps provided by Nextcloud, you can use other apps to
```

I would prefer not to list ownCloud unless the apps provided by ownCloud are explicitly tested against the recent Nextcloud releases.